internal/konnect/client.go

```diff
@@ -0,0 +1,169 @@
+package konnect
+
+import (
+	"bytes"
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/kong/kubernetes-ingress-controller/v2/internal/adminapi"
+	tlsutil "github.com/kong/kubernetes-ingress-controller/v2/internal/util/tls"
+)
+
+type KonnectAdminClient struct {
+	Address        string
+	RuntimeGroupID string
+	Client         *http.Client
+}
+
+func NewKonnectAdminClient(cfg adminapi.KonnectConfig) (*KonnectAdminClient, error) {
```

Comment:
This could also be called `NewAdminClient`

Plus, please add a doc comment for this func.

Reply:
done